def/ar/about.cson

```diff
@@ -1,15 +1,15 @@
-About:
-  tabTitle: "عن"
-  releaseNotes: "ملاحظات الإصدار"
-  newUpdate: "تحديث جديد"
-  upToDate: "اتوم محدث لآخر إصدار!"
-  checkingForUpdates: "يجري التحقق من التحديثات..."
-  checkUpdatesNow: "تحقق الآن"
-  automaticDownloadUpdates: "تنزيل التحديثات آليا"
-  restartInstall: "أعد التشغيل وثبت"
-  license: "الرخصة"
-  termsOfUse: "شروط الإستخدام"
-  with: " بـ"
-  by: " من قبل "
-  andTheAwesome: "والرائع "
-  atomCommunity: "مجتمع أتوم"
+About:
+  tabTitle: "عن"

```

Comment:
Looks like there are some whitespace changes, please help remove these, thanks!

Reply:
I am sorry to reply so late. I do not understand the situation, because I have never pushed `def/ar/about.cson` yet.
And `def/ar/about.cson` in my local repository is the just the same as that of atom-i18n version 0.25.0, with WinMerge tools.
I noticed that some source code (*.js) and json files used LF as the line separator, and now, use CRLF as it.
So, the files that I pushed as Pull Request use CRLF.  Should I push `def/ar/about.cson` with CRLF?

Reply:
@juggernautjp You may just merge the latest `main` branch and push, to update this PR.

```
# update your local main branch
git checkout main
git pull

# update your local PR branch (merge with main)
git checkout add/localize-welcome-views
git merge main --no-edit

# push PR branch
git push
```